docker/docker.go

```diff
@@ -357,9 +374,39 @@ func (d *docker) CollectMetrics(mts []plugin.MetricType) ([]plugin.MetricType, e
 
 // GetConfigPolicy returns plugin config policy
 func (d *docker) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	return cpolicy.New(), nil
+	policy := pluginlib.NewConfigPolicy()
```

Comment:
@kindermoumoute let's split work into two pieces:
 - switch to new lib
 - use config-based Docker endpoint

Thoughts?